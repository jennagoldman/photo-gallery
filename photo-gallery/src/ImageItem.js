import React, { Component } from 'react';
import imagesData from './new-data';

export default class ImageItem extends Component {
    render() {
        return (
            <li className="image-container">
                <img src={this.props.img.url} alt={this.props.img.title} keyword={this.props.img.keyword} />
            </li>
        )
    }
}