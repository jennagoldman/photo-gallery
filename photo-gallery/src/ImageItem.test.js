import React from 'react';
import ImageItem from './ImageItem';
import images from './new-data';
import renderer from 'react-test-renderer';

it('ImageItem renders correctly', () => {
  const tree = renderer
    .create(<ImageItem img={images} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});