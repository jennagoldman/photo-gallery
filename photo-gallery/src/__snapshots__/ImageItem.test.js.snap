// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageItem renders correctly 1`] = `
<li
  className="image-container"
>
  <img />
</li>
`;
